### DISCUSSION TITLE
Adding second email

### DISCUSSION THREAD

**Discussion by resanwilda** [community/community#135640] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have the following message in my email settings: "You have a single verified email associated with your GitHub account. Add an additional verified email address in case you lose access to your primary email." I added a second email and it sent a verification request to that email. It asked me to login to Github. I didn't think I had a Github account set up for that email so I went to create one but it said there WAS an account for the second email. I tried logging in hoping it would remember my password but it didn't. So I did "Reset Password" and it said that there is no Github account associated with that email.

**Comment by eabdalmufid** [comment, marked as answer]
I figured out the problem. It wanted me to login with my main account to verify the second account. Did that and it worked.
